rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/api/BakeOptions.groovy

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2015 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.rosco.api
+
+import groovy.transform.InheritConstructors
+
+class BakeOptions {
+
+  String cloudProvider
+  List<BaseImage> baseImages
+
+  static class BaseImage {
+    String id
+    String shortDescription
+    String detailedDescription
+    BakeRequest.PackageType packageType
+  }
+
+  static class Selected {
```

Comment:
Is this necessary? We can't just hold a reference to `BakeOptions.BaseImage` directly in places where `BakeOptions.Selected` is now used?